src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -311,6 +312,12 @@ private static object GetTypedProperty(IPublishedContent content, Type type)
                                 }
                             }
                         }
+                        else if (propertyInfo.PropertyType == typeof(HtmlString))
+                        {
+                            // Handle Html strings so we don't have to set the attribute.
+                            HtmlStringConverter converter = new HtmlStringConverter();
+                            propertyInfo.SetValue(instance, converter.ConvertFrom(propertyValue), null);
```

Comment:
Cool.  At the moment I'm adding this to a custom `ApplicationStarted` event:

    TypeDescriptor.AddAttributes(typeof(HtmlString), new TypeConverterAttribute(typeof(HtmlStringConverter)));

YEY! I can remove some of my code! :scissors: